chain/yoda/handler.go

```diff
@@ -218,27 +218,31 @@ func handleRawRequests(c *Context, l *Logger, id types.RequestID, reqs []rawRequ
 }
 
 func handleRawRequest(c *Context, l *Logger, req rawRequest, key keys.Info, id types.RequestID, processingResultCh chan processingResult) {
+	c.updateHandlingGauge(1)
```

Comment:
Can we use defer c.updateHandlingGauge(-1) for this use case?

Reply:
Updated